src/components/Grid/Grid.md

```diff
@@ -26,7 +26,7 @@ import { Grid } from './Grid.js'
 import { Row } from './Row.js'
 import { Column } from './Column.js'
 import { useGridSorting } from './useGridSorting.js'
-import styles from './grid-example.module.css'
+import styles from './grid-example.module.scss'
```

Comment:
Fwiw, I elected to just rename all `src/components/Grid/*` css files to .scss for consistency sake even though there were only a couple using `composes:` and confusing monorepo. My thought was it was better within this directory at least to be consistent